doc/build-cross.md

```diff
@@ -35,59 +35,71 @@ When building Dash Core, use
 $ ./configure --prefix=`pwd`/depends/x86_64-apple-darwin11
 ```
 
-Windows 64bit/32bit Cross-compilation
+Windows 64bit Cross-compilation
 -------------------------------
-Cross-compiling to Windows requires a few additional packages to be installed:
+The steps below can be performed on Ubuntu (including in a VM) or WSL. The depends system
+will also work on other Linux distributions, however the commands for
+installing the toolchain will be different.
 
-```bash
-$ sudo apt-get install nsis wine-stable wine64 bc
-```
+First, install the general dependencies:
 
-For Windows 64bit, install :
-```bash
-$ sudo apt-get install g++-mingw-w64-x86-64
-```
+    sudo apt update
+    sudo apt upgrade
+    sudo apt install build-essential libtool autotools-dev automake pkg-config bsdmainutils curl git python3 cmake     
 
-If you're building on Ubuntu 17.04 or later, run these two commands, selecting the 'posix' variant for both,
-to work around issues with mingw-w64. See issue [8732](https://github.com/bitcoin/bitcoin/issues/8732) for more information.
-This also fixes linker issues related to std::thread and other threading related standard C++ libraries.
-```
-sudo update-alternatives --config x86_64-w64-mingw32-g++
-sudo update-alternatives --config x86_64-w64-mingw32-gcc
-```
+A host toolchain (`build-essential`) is necessary because some dependency
+packages need to build host utilities that are used in the build process.
 
-For Windows 32bit, install:
-```bash
-$ sudo apt-get install g++-mingw-w64-i686
-```
+See [dependencies.md](dependencies.md) for a complete overview.
 
-If you're building on Ubuntu 17.04 or later, run these two commands, selecting the 'posix' variant for both,
-to fix linker issues related to std::thread and other threading related standard C++ libraries.
-```
-sudo update-alternatives --config x86_64-w64-mingw32-g++
-sudo update-alternatives --config x86_64-w64-mingw32-gcc
-```
+If you want to build the windows installer with `make deploy` you need [NSIS](https://nsis.sourceforge.io/Main_Page):
 
-Before building for Windows 32bit or 64bit, run
+    sudo apt install nsis
 
-```
-$ PATH=$(echo "$PATH" | sed -e 's/:\/mnt.*//g') # strip out problematic Windows %PATH% imported var
-```
+Acquire the source in the usual way:
 
-When building the dependencies, as described in [build-generic](build-generic.md), use
+    git clone https://github.com/dashpay/dash.git
+    cd dash
 
-```bash
-$ make HOST=x86_64-w64-mingw32 -j4
-```
+## Building for 64-bit Windows
```

Comment:
Let's make this header a bit smaller to show that it belongs to `Windows 64bit Cross-compilation` section
```suggestion
### Building for 64-bit Windows
```